cmd2/cmd2.py

```diff
@@ -3438,9 +3437,21 @@ def do_eos(self, _: argparse.Namespace) -> None:
     load_description = ("Run commands in script file that is encoded as either ASCII or UTF-8 text\n"
                         "\n"
                         "Script should contain one command per line, just like the command would be\n"
-                        "typed in the console.")
+                        "typed in the console.\n"
+                        "\n"
+                        "It loads commands from a script file into a queue and then the normal cmd2\n"
+                        "REPL resumes control and executes the commands in the queue in FIFO order.\n"
+                        "If you attempt to redirect/pipe a load command, it will capture the output\n"
+                        "of the load command itself, not what it adds to the queue.\n"
```

Comment:
Probably show the user they can add redirection to the commands being run in the script file.
ex: `command blah blah > file.txt`

This also applies to the change made in freefeatures.rst.

Reply:
I'll try to add something appropriate